libs/text-splitters/langchain_text_splitters/html.py

```diff
@@ -1,16 +1,76 @@
+'''
+This module provides classes for splitting HTML content into structured
+Document objects based on specified header tags.
+
+    ElementType(TypedDict): A typed dictionary representing an element type.
```

Comment:
We don't want to document any of these -- the API reference does this automatically by scanning the code. Poke around the code base e.g. look at the langchain core package to see how things are typically documented.

We follow Google document style for docstrings but without documenting the type as it's already documented via type hints